packages/@aws-cdk/integ-tests-alpha/lib/assertions/private/deploy-assert.ts

```diff
@@ -59,7 +59,12 @@ export class DeployAssert extends Construct implements IDeployAssert {
   }
 
   public awsApiCall(service: string, api: string, parameters?: any, outputPaths?: string[]): IApiCall {
-    return new AwsApiCall(this.scope, `AwsApiCall${service}${api}`, {
+    let hash = 'NoParameter';
```

Comment:
```suggestion
    let hash = '';
```

Can we default to no hash instead.

Reply:
Yes, sure, and it's simpler! I've just pushed changes.